iterations_runners/iterations_runner.c

```diff
@@ -15,11 +15,14 @@
 #include <dlfcn.h>
 #include <err.h>
 #include <string.h>
+#include <inttypes.h>
 
 #define BENCH_FUNC_NAME "run_iter"
+#define TSR_DEFAULT 0xffffffffffffffff
```

Comment:
`0` would be more sensible here, surely? [Especially if we sometimes use doubles.]